api/admin/admin-announcements/bundle.md

```diff
@@ -0,0 +1,56 @@
+# admin-announcements
+
+Admin bundle for making announcements.
+
+## Description
+
+The bundle allows admins to create announcements to be displayed in announcements tab. Admins can optionally also set the announcement to be displayed in a pop-up. 
+
+## Announcement parameters
+
+These are the parameters you can set for an announcement when creating a new announcement or editing an old one.
+
+<table class="table">
+  <tr>
+    <th> Parameter </th><th> Functionality</th>
+  </tr>
+  <tr>
+    <td>Title</td><td>Title of the announcement. (String)</td>
+  </tr>
+  <tr>
+    <td>Content</td><td>The body of the announcement. (String)</td>
+  </tr>
+  <tr>
+    <td>Starting date</td><td>Starting date of the announcement. Announcement won't be displayed until its starting date. This is displayed as "Valid" date selector. (Date)</td>
+  </tr>
+  <tr>
+    <td>End date</td><td>Ending date of the announcement. Announcement won't be displayed after its ending date. This is displayed as "Valid" date selector. (Date)</td>
+  </tr>
+  <tr>
+    <td>Active</td><td>If toggled on, the announcement will show as a pop-up when users enter the application. The pop-up can be toggled off by the users after they have seen it. (Boolean)</td>
+  </tr>
+</table>
+
+## TODO
+
+* Make Admin-announcements bundle to be a separate tab on the same tile OR make the functionality be similar to admin-layereditor.
+* Finish DeleteButton.js
```

Comment:
The DeleteButton.js was removed so I think this TODO can be removed as well.

Reply:
removed